src/python/WMCore/Services/Requests.py

```diff
@@ -305,8 +305,6 @@ def decodeResult(self, result, decoder):
             result = decoder(result)
         elif decoder is not False:
             result = self.decode(result)
-        if PY3:
-            result = decodeBytesToUnicode(result)
```

Comment:
This change is ok, but we should keep in mind that in py3 we may incur in problems if we call `makeRequest` with `decoder=False`. Maybe it's worth to add such a comment to `makeRequest` docstring, or somthing like that

Reply:
That's a good idea. I'm adding a comment to the docstring. But I assume that anyone setting it to False, actually knows what he's doing (since the default value is True).